### ISSUE TITLE
Netlify fuer Previews?

### ISSUE THREAD

**Issue by stkdiretto** [okfde/buendnis-f5#10]
Die Buildprozesse brechen immer wieder mal, wenn man neue Posts einfach „live“ im Main-Branch reinbaut und dann steht schon was und ist dann kap0tt. Loesungsansatz 1: Immer lokal mit hugo testen. Sehe ich nicht passieren :D Loesungsansatz 2: Branch machen, build preview mit netlify und dann mergen. Auch semioptimal glaub ich.

@greg0r, ideen?

**Comment by greg0r** [comment]
Du meinst diese failure builds?
https://github.com/okfde/buendnis-f5/actions?query=is%3Afailure+
sieht für mich recht entspannt aus, oder übersehe ich was?

**Comment by stkdiretto** [comment]
Ja, es ist kein Beinbruch. Problem ist nur, wenn irgendwas h akt (Vertipper im Filename, nicht eingebundene Bilder o.ae.) und niemand von den Leuten mit hugo- und git-Ahnung gerade verfuegbar ist weil Urlaub oder so, um das zu fixen. Option 1 waere, dann ist das halt so und bidde keine Fehler machen. Ich kenn's aber halt so, dass man neue Posts mit Branch und PR macht und netlify dann Vorschau erlaubt. Aber vielleicht ist das auch Overkill. Wollte vor allem ne zweite Meinung haben ^^ Wenn du sagst, das passt schon so, einfach Issue zumachen und wir machen das weiter wie bisher und ich versuch bei den Stellen, wo ich's kann, bissel mehr die Prozesse zu erklaeren.
